retryablemessageprocessor.go

```diff
@@ -14,16 +14,16 @@ import (
 // MaxRetries is the maximum number of time's we'd like to retry processing a record before quitting
 const consumerRetriesExceeded = "kinesis.consumer_error.retries_exceeded"
 
-// MaxRetriesExceededError implements MessageProcessError and is used to indicate to upstream application/
+// MaxRetriesExceededError implements MessageProcessorError and is used to indicate to upstream application/
 // decorators that retries have been attempted and exhausted
 type MaxRetriesExceededError struct {
 	Retryable bool
 	OrigErr   error
 	Wait      int
 }
 
-func (t MaxRetriesExceededError) Error() error {
-	return t.OrigErr
+func (t MaxRetriesExceededError) Error() string {
+	return t.OrigErr.Error()
```

Comment:
question: Why not prefix the original message with something like "max retries exceeded: " since this message is all that will show up in the logs?

Reply:
oops

Reply:
👍